assets/aa_video.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.0

```

Comment:
This doesn't have to be here right? I would think this should only exist within OpenAuto

Reply:
OpenAuto's resource pack isn't built with IA, as the OA resource pack is only built into the standalone autoapp - having it in both should just preserve compatibility.